artifacts/debian/debian.go

```diff
@@ -0,0 +1,117 @@
+package debian
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/containerdisks/pkg/api"
+	"kubevirt.io/containerdisks/pkg/architecture"
+	"kubevirt.io/containerdisks/pkg/docs"
+	"kubevirt.io/containerdisks/pkg/hashsum"
+	"kubevirt.io/containerdisks/pkg/http"
+	"kubevirt.io/containerdisks/pkg/tests"
+)
+
+type debian struct {
+	Arch            string
+	Version         string
+	VersionName     string
+	getter          http.Getter
+	ExampleUserData *docs.UserData
+	envVariables    map[string]string
+}
+
+const (
+	baseURLFmt   = "https://cloud.debian.org/images/cloud/%s/latest/"
+	baseQcow2Fmt = "debian-%s-genericcloud-%s.qcow2"
+	description  = `Debian Generic Cloud images for KubeVirt.
+<br />
+<br />
+Visit [debian.org](https://cloud.debian.org/images/cloud/) to learn more about Debian project.`
+)
+
+func (d *debian) Metadata() *api.Metadata {
+	metadata := &api.Metadata{
+		Name:            "debian",
+		Version:         d.Version,
+		Description:     description,
+		ExampleUserData: *d.ExampleUserData,
```

Comment:
L41 is duplicted by the block starting at L44. L41 should be dropped.

Reply:
Done.